### PR TITLE
ADJUST1-450 DR: Review remand details

### PR DESCRIPTION
**Pull request by pavilionsahota**
Renamed labels // Removed bullet points from offences // Added total days component to review screen

### CODE REVIEW

File: server/model/remandReviewModel.ts [hunk 1]
```diff
@@ -61,16 +77,29 @@ export default class RemandReviewModel {
             text: 'Offences',
           },
           value: {
-            html: `<ul class="govuk-list govuk-list--bullet">
+            html: `<div class="govuk-list">
```
Comment (ldlharper) [inline server/model/remandReviewModel.ts:80]: Should this be a div with list class? I'm not sure thats how the govuk-list style should be used.

File: server/model/remandReviewModel.ts [hunk 2]
```diff
@@ -61,16 +77,29 @@ export default class RemandReviewModel {
...
                     ${offences
-                      .map(
-                        it =>
-                          `<li>${it.offenceDescription}${
-                            it.recall ? '<strong class="govuk-tag">Recall</strong>' : ''
-                          }</li>`,
-                      )
+                      .map(it => {
+                        let committedText
```
Comment (ldlharper) [inline server/model/remandReviewModel.ts:83]: Can you move this committed text logic into another function?

File: server/model/remandReviewModel.ts (hunk 1 above)
Comment (pavilionsahota) [reply server/model/remandReviewModel.ts:80]: I've removed the classes as they are no longer used.

**Review by ldlharper — APPROVED** [review]
LGTM

**Pull request merged**